libs/ui/src/ListBox/ListBox.tsx

```diff
@@ -9,7 +10,7 @@ type Props = PropsWithChildren<HTMLAttributes<HTMLUListElement>>
 
 export const ListBoxRoot = forwardRef(
   ({ children, className, ...restProps }: Props, ref: ForwardedRef<HTMLUListElement>) => (
-    <ul {...restProps} ref={ref} className={`${styles.list} ${className ?? ''}`}>
+    <ul {...restProps} ref={ref} className={clsx(styles.list, className)}>
```

Comment:
We can simplify this test, we don't have to test for undefined any more. The `clsx` package handles that.